parsers/yamlparser_test.go

```diff
@@ -26,14 +26,13 @@ import (
 
 var manifestfile_val_pkg = "../tests/dat/manifest_validate_package_grammar.yaml"
 var manifestfile_val_tar = "../tests/dat/manifest_validate_trigger_action_rule_grammar.yaml"
-var manifestfile3 = "../tests/dat/manifest3.yaml"
-var manifestfile4 = "../tests/dat/manifest4.yaml"
-var manifestfile5 = "../tests/dat/manifest5.yaml"
-var manifestfile6 = "../tests/dat/manifest6.yaml"
+var manifestfile3 = "../tests/dat/manifest_validate_triggerfeed.yaml"
```

Comment:
Can we also rename the variables in the package as well; for example

rename 'manifestfile4' to 'manifestfile_validate_rule' which makes the code below that uses the variable able to conform that the variable does indeed match the test.  

This would be the same comment for manifestfile4 through 6 and testfile3 through 4.